Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -301,5 +309,157 @@ configuring the CiliumLocalRedirectPolicy.
+      $ helm repo update
+      $ helm template kiam uswitch/kiam > kiam.yaml
+
+  - If you see an error like "request blocked by whitelist-route-regexp" while
+    running requests to the metadata server, then you may need to whitelist the
+    metadata requests by passing the below argument to the ``kiam-agent Deamonset``.
+
+    .. code-block:: bash
+
+        $ sed -i '/args:/a \ \ \ \ \ \ \ \ \ \ \ \ - --whitelist-route-regexp=meta-data' kiam.yaml
+
+
+  - Make sure the "--iptables" argument is removed from the arguments passed
+    to the ``kiam-agent Daemonset``.
+
+  - Make sure the ``kiam-agent Daemonset`` is run in the ``hostNetwork`` mode.
+
+  - Apply the kiam configuration.
+
+    .. code-block:: bash
+
+        $ kubectl apply -f kiam.yaml
+
+
+- Deploy the Local Redirect Policy to redirect pod traffic to the deployed kiam agent.
+
+  .. parsed-literal::
+
+      $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/kiam-lrp.yaml
+
+.. note::
+
+    - The ``addressMatcher`` ip address in the Local Redirect Policy is set to
+      the ip address of the AWS metadata server and the ``toPorts`` port
+      to the default HTTP server port. The ``toPorts`` field under
+      ``redirectBackend`` configuration in the policy is set to the port that
+      the kiam agent listens on. The port is passed as "--port" argument in
+      the ``kiam-agent Daemonset``.
+    - The Local Redirect Policy namespace is set to the namespace
+      in which kiam-agent Daemonset is deployed.
+
+- Once all the kiam agent pods are in ``Running`` state, the metadata requests
+  from application pods will get redirected to the node-local kiam agent pods.
+  You can verify this by running a curl command to the AWS metadata server from
+  one of the application pods, and tcpdump command on the same EKS cluster node as the
+  pod. Following is an example output, where ``192.169.98.118`` is the ip
+  address of an application pod, and ``192.168.33.99`` is the ip address of the
+  kiam agent running on the same node as the application pod.
+
+  .. code-block:: bash
+
+      $ kubectl exec app-pod -- curl -s -w "\n" -X GET http://169.254.169.254/latest/meta-data/
+
+  .. code-block:: bash
+
+      $ sudo tcpdump -i any -enn "(port 8181) and (host 192.168.33.99 and 192.168.98.118)"
+      tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
+      listening on any, link-type LINUX_SLL (Linux cooked), capture size 262144 bytes
+      05:16:05.229597  In de:e4:e9:94:b5:9f ethertype IPv4 (0x0800), length 76: 192.168.98.118.47934 > 192.168.33.99.8181: Flags [S], seq 669026791, win 62727, options [mss 8961,sackOK,TS val 2539579886 ecr 0,nop,wscale 7], length 0
+      05:16:05.229657 Out 56:8f:62:18:6f:85 ethertype IPv4 (0x0800), length 76: 192.168.33.99.8181 > 192.168.98.118.47934: Flags [S.], seq 2355192249, ack 669026792, win 62643, options [mss 8961,sackOK,TS val 4263010641 ecr 2539579886,nop,wscale 7], length 0
 
+Miscellaneous
```

Comment:
:thinking: can we combine miscellaneous + limitations?

Reply:
I don't see any overlap so better to keep them separate?